src/server/controller/cycleData/persistNodeValue/expressionEvaluator/functions/validatorSubCategory.ts

```diff
@@ -13,6 +13,7 @@ export const validatorSubCategory: ExpressionFunction<Context> = {
     return (categoryValue?: string, subCategoryValues?: Array<string>): NodeValueValidation => {
       const valid =
         Objects.isEmpty(categoryValue) ||
+        !subCategoryValues?.every(Boolean) ||
```

Comment:
does it work when category is empty and every subCategoryValues is true ?

Reply:
Yes, because of the first check?
`Objects.isEmpty(categoryValue) ||`

![image](https://user-images.githubusercontent.com/5508251/197323525-5b5429ff-d36f-4b77-b079-f9886b628041.png)

Reply:
missed that 😁